Docs/Architecture/CommandBased.md

```diff
@@ -2,26 +2,47 @@
 
 ## Command Based is a way of structuring our code to make it easy to do complicated tasks, and write control logic in a concise way
 
-Command based programming revolves around two concepts: Subsystems and Commands.
+Command based programming revolves around three concepts: **Subsystems**, **Commands**, and **Triggers**.
+
 A Subsystem is a set of hardware that forms one system on our robot, like the drivetrain, elevator, arm, or intake.
 Each subsystem contains some associated hardware (motors, pistons, sensors, etc.) They are the "nouns" of our robot, what it is.
+Each Subsystem is generally made to contain as broad of a set of hardware that will always operate as a unit as possible.
```

Comment:
"as possible" can probably be dropped here? Unless you mean 'if' but would be kind of weird to have parts of a subsystem not working in concert? Expert level choice to work around that and pretty specific circumstances.